src/main/java/com/commafeed/backend/feed/FeedParser.java

```diff
@@ -134,8 +134,8 @@ private void handleForeignMarkup(SyndFeed feed) {
 			}
 		}
 	}
-
-	private Date getEntryUpdateDate(SyndEntry item) {
+	// This function have been changed to protected for testing purposes

```

Comment:
I'm not sure we should change the visibility of methods when making tests. I'm not 100% sure but I think that simply testing the public methods and finding whether or not the result is as expected is good enough. Who cares what the private helper methods returns, as long as the public method works well.

Reply:
I will get it as a question for the teacher next class...